tests/Api/Response/Parser/FiltersParserTest.php

```diff
@@ -1119,4 +1135,160 @@ public function parseDataProvider()
             ],
         ];
     }
+
+    /**
+     * @dataProvider parseForWidgetsDataProvider
+     */
+    public function testParseForWidgets(array $parsedFilters, array $expectedResult)
+    {
+        $xml = new SimpleXMLElement('<filters/>');
+        /** @var FiltersParser|MockObject $filtersParserMock */
+        $filtersParserMock = $this->getFiltersParserMock(['parse']);
+        $filtersParserMock->method('parse')->willReturn($parsedFilters);
+        $this->assertEquals($expectedResult, $filtersParserMock->parseForWidgets($xml));
+    }
+
+    public function parseForWidgetsDataProvider(): array
+    {
+        return [
+            'typesGetMappedCorrectlyIfValuesExist' => [
+                [
+                    [
+                        'id' => 'vendor',
+                        'name' => 'Manufacturer',
+                        'type' => 'image',
+                        'values' => [
+                            ['id' => 1]
```

Comment:
Similar to `parseDataProvider`, shouldn't this contain more as just the id? Please try to use more realistic data.

Reply:
It does contain more than just the ID but it really isn't relevant for the test and the full objects are quite large. Do I just copy everything over?

Reply:
Each filter containing 1-2 values should be sufficient.